docs/src/modules/components/branding/Quote.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import t1 from './t1';
+
+const Quote = styled((props) => {
+  const { author = {}, children, ...other } = props;
+  return (
+    <Grid container {...other}>
+      <Box
+        component="img"
+        src="/static/images/branding/quote.svg"
+        sx={{ width: 32, height: 24, marginBottom: 2 }}
+        alt={t1('Quote')}
+      />
+      <Typography variant="h4">&quot;{children}&quot;</Typography>
+      <Grid container spacing={2} alignItems="center">
+        <Grid item>
+          <Avatar src={author.avatar} />
+        </Grid>
+        <Grid item>
+          <Typography>
+            {author.name}
+            {', '}
+            {author.title}
+          </Typography>
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+})({
+  background: '#F3F6F9',
```

Comment:
I think that we can start with the names used in the Sketch file and later rename them to something we prefer.

```suggestion
  backgroundColor: theme.palette.greyF3,
```